src/uploadx/lib/options.ts

```diff
@@ -30,6 +30,16 @@ export interface UploadxOptions extends UploaderOptions {
    * @defaultValue true
    */
   multiple?: boolean;
+  /**
+   * Keep an incomplete upload url to allow resuming after browser restart
+   * @deprecated use storeIncompleteHours = 0 to disable
+   */
+  storeIncompleteUploadUrl?: boolean;
+  /**
+   * Retention time for incomplete uploads
+   * @defaultValue 24
+   */
+  storeIncompleteHours?: number;
```

Comment:
@siovene 
`storeIncompleteHours: 0`  is equivalent to `storeIncompleteUploadUrl: false`. 
 I am thinking of removing `storeIncompleteUploadUrl` ?

Reply:
@kukhariev makes sense. Just please remember that if you break the API, you need to bump the major version, so you might want to keep this deprecated and working until you want to bundle more major changes into a major version bump.